src/TourManager/Api/Controllers/TourController.cs

```diff
@@ -4,13 +4,47 @@
 namespace Travely.TourManager.Api.Controllers
 {
     [ApiController]
-    [Route("[controller]")]
+    [Route("api/v1/[controller]")]
     public class TourController : Controller
     {
+        // GET: api/v1/tour
         [HttpGet]
+        [Route("")]
         public async Task<IActionResult> Get()
         {
             return Ok();
         }
+
+        // GET: api/v1/tour/{id}
+        [HttpGet]
+        [Route("{id}")]
```

Comment:
You can specify route parameters in HttpGet attribute

Reply:
I'm not sure the latest changes are applicable for post/put/delete requests. have a look please.